index.js

```diff
@@ -28,6 +34,13 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
+function drinkWater(current_thirst_level) {
+  console.log("Man I sure am thirsty");
+  current_thirst_level --;
+  console.log("Ahh that hits the spot");
+  return current_thirst_level;
+}
```

Comment:
Looks good, just a minor nitpick - variable names in Javascript are typically camelCased, so `current_thirst_level` should be `currentThirstLevel`.